contrib/nsqadmin.cfg.example

```diff
@@ -1,6 +1,42 @@
 ## log verbosity level: debug, info, warn, error, or fatal
 log_level = "info"
 
+## log message prefix (default "[nsqadmin] ")
+# log_prefix = ""
+
+## HTTP header to check for authenticated admin users (default "X_Forwarded_User")
+# acl_http_header = ""
+
+## admin user (may be given multiple times; if specified, only these users will be able to perform privileged actions; acl_http_header is used to determine the authenticated user)
```

Comment:
this help text doesn't work exactly as-is for this config format ("given multiple timpes"?)

also it should be wrapped somewhere around 80 chars depending on taste ;)